bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -283,6 +283,10 @@ private void applyConfig(@Nullable Map<String, Object> config) {
             }
             if (!isReadOnly && hasStateOptions(itemName)) {
                 return SitemapFactory.eINSTANCE.createSelection();
+            }
+            Set<String> tags = getItemTags(itemName);
+            if (!isReadOnly && NumberItem.class.isAssignableFrom(itemType) && tags.contains("Setpoint")) {
+                return SitemapFactory.eINSTANCE.createSetpoint();
```

Comment:
Wouldn't it be better in line with the other methods to have `hasItemTag(itemName, "Setpoint")`?

Reply:
Yes, especially when a method `hasTag` is available in Item class. 
I will change that.

Reply:
Done